### PR TITLE
issue #13 Fixed:"Back to top" button should not be displayed when we …

### LINKED ISSUE #13: "Back to top" button should not be displayed when we are already at top.
Algo_phantoms

### PR DESCRIPTION
**Pull request by shitesh2607**
# Description
 "Back to top" button should not be displayed when we are already at top.
 Fixes #13

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before : 
106865622-71f58780-66f1-11eb-9211-7e34eb16e4cb

    - After :
ScrShot

## Checklist:

- My code follows the style guidelines of this project
- I have performed a self-review of my own code
-  I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -2,6 +2,9 @@
 @import url(https://weloveiconfonts.com/api/?family=entypo);
 
 /******************* PRELOADER ******************/
+html{
+    scroll-behavior: smooth;
```
Comment (harikesh409) [inline css/style.css:6]: It is already present for `*` property so adding again is just a repetition.

File: js/index.js [hunk 2]
```diff
@@ -3,13 +3,15 @@
...
-        backToTopButton.style.display = "none";
+        backToTopButton.classList.remove("active");
     }
 });
 
 backToTopButton.addEventListener("click", () => {
     window.scrollTo(0,0);
 });
+
+con
```
Comment (harikesh409) [inline js/index.js:17]: remove unnecessary line.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by shitesh2607** [comment]
sure

**Comment by shitesh2607** [comment]
Hi @harikesh409 I have updated as per requested changes. So now review it that I can close I this issue.

**Review by harikesh409 — APPROVED** [review]

**Pull request merged**